plugins/module_utils/mysql.py

```diff
@@ -132,16 +132,12 @@ def mysql_connect(module, login_user=None, login_password=None, config_file='',
                 version)
 
 
-def mysql_common_argument_spec():
-    return dict(
+def proxysql_common_argument_spec():
+    return = dict(
         login_user=dict(type='str', default=None),
         login_password=dict(type='str', no_log=True),
-        login_host=dict(type='str', default='localhost'),
-        login_port=dict(type='int', default=3306),
+        login_host=dict(type='str', default='127.0.0.1'),
+        login_port=dict(type='int', default=6032),
```

Comment:
```suggestion
        login_host=dict(type='str', default='127.0.0.1'),
        login_port=dict(type='int', default=6032),
```
I've just noticed these changes (plus config_file a bit below). These are breaking ones.

If it's really really necessary:
1. We should create a PR announcing (under `major_changes:`) that this is coming in the next major release and adding warning when the defaults for these parameters are used that they will be changed in `2.0.0`.
2. Plan a major release in several months (better in half a year).
3. Create a separate PR that will change the defaults and merge it right before releasing `2.0.0`.

Reply:
It's not a breaking change, becase this function was never used in any module. It was just copy pasted from community.mysql when the split happens into the single collecitons.

Reply:
...and I adjust the default values in mysql.py as they are set in the single modules.

Reply:
for example: https://github.com/ansible-collections/community.proxysql/pull/64/files#diff-a2f0a1be2d5ebd2786bd069a42c54c50b5fb4c97effaa7a13b3658f0ec6cb849L421

Reply:
Ah, clear now:)) Thanks:)